opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml

```diff
@@ -272,10 +232,9 @@ components:
         format: uuid
 
     X-XSRF-TOKEN:
+      name: Redirect-Code
```

Comment:
Can we use same name and declaration please, it is confusing to have redirectCode and `X-XSRF-TOKEN` both

Reply:
changed header name to "X-XSRF-TOKEN"